contracts/SSVNetwork.sol

```diff
@@ -146,8 +146,8 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
                 ++dao.validatorCount;
                 _dao = dao;
             }
-
-            require(!_liquidatable(pod.balance, pod.validatorCount, operatorIds), "account liquidatable");
+            // IMPROPER ERROR SCHEMA ALSO WHY NOT IN  _validateValidatorParams FUNCTION
```

Comment:
@andrew-blox there are different types of validation.
- validation for the input func params
- business logic validation (for instance, is liquidatable or not)

thats not good idea to merge them into one place.